modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/evict/Random2LruPageEvictionTracker.java

```diff
@@ -69,7 +69,7 @@ public Random2LruPageEvictionTracker(
 
     /** {@inheritDoc} */
     @Override public void start() throws IgniteException {
-        trackingArrPtr = GridUnsafe.allocateMemory(trackingSize * 8);
+        trackingArrPtr = GridUnsafe.allocateMemory((long)trackingSize * 8);
```

Comment:
A long constant can be used instead of casting for better readability: `8L`, `4L`